ppr-ui/src/components/mhrRegistration/ReviewConfirm/SubmittingPartyReview.vue

```diff
@@ -16,34 +16,142 @@
         </span>
       </section>
 
-      <!-- divider -->
-      <div class="px-4">
-        <v-divider />
-      </div>
-
       <!-- -->
       <template>
         <section class="pt-6" id="review-submitting-party-section">
           <!-- Insert Review mode of component here -->
+          <v-row no-gutters class="px-6 pb-7">
+            <v-col cols="3">
+              <h3 class="headers">Name</h3>
+            </v-col>
+            <v-col cols="3" class="pl-1">
+              <h3 class="headers">Mailing Address</h3>
+            </v-col>
+            <v-col cols="3" class="pl-3">
+              <h3 class="headers">Email Address</h3>
+            </v-col>
+            <v-col cols="3" class="pl-4">
+              <h3 class="headers">Phone Number</h3>
+            </v-col>
+          </v-row>
+
+          <div class="px-4">
+            <v-divider />
+          </div>
+
+          <v-row no-gutters class="px-6 py-7">
+            <v-col cols="3">
+              <v-row no-gutters>
+                <v-col cols="1" class="mr-2">
+                  <v-icon color="black">
```

Comment:
Would double check icon colour vs designs. I'm not sure we use plain black too often but correct me if i'm wrong!